src/Camera.tsx

```diff
@@ -261,6 +309,12 @@ export class Camera extends React.PureComponent<CameraProps> {
    * ```
    */
   public async stopRecording(): Promise<void> {
+    this.setState(({ timestamps: prevTimestamps }) => ({
+      timestamps: {
+        ...prevTimestamps,
+        requestRecordingEndedAt: new Date().valueOf(),
+      },
```

Comment:
Need to think this one through as to whether it would be accepted into the library.  Is there any other way we can return the timestamps?  Metadata on the video itself maybe?

Reply:
yes I will make them return inside metadata props.